docs/html/class_c_a_r_o_m_1_1_d_m_d-members.html

```diff
@@ -0,0 +1,113 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
```

Comment:
What is the decision on this? When I build, git shows lots of modified html and js files. Should these be in the repo?

Reply:
The html and js files are needed, but perhaps they don't need to be pushed as often. Let me see if there is a way to only have it show up when the content of the webpage has actually changed rather than unimportant metadata.